oktapam/resource_security_policy.go

```diff
@@ -164,6 +207,56 @@ func resourceSecurityPolicy() *schema.Resource {
 							Description: descriptions.Privileges,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									attributes.Secret: {
+										Type:        schema.TypeList,
+										Optional:    true,
+										MaxItems:    1,
+										Description: descriptions.PrivilegeSecret,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												attributes.List: {
```

Comment:
Why these privileges need to be required ? Also, there are different privileges for secret and folder. If in the policy, we have a SecretFolder resource then what is the relevance of PrivilegeSecret*.

Reply:
My goal here is to be explicit on the privileges granted, even if the default is false.  

There are not different privileges for a secret/folder.